internal/cmd/operator-sdk/olm/install.go

```diff
@@ -35,8 +35,6 @@ func newInstallCmd() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&mgr.Version, "version", olm.DefaultVersion, "version of OLM resources to install")
-	cmd.Flags().StringVar(&mgr.OLMNamespace, "olm-namespace", olm.DefaultOLMNamespace,
```

Comment:
`mgr.OLMNamespace` is still used in `InstallVersion`. We should remove this field from the manager's struct and pass the hard-coded default `olm`.

Reply:
`mgr.OLMNamespace` is still used by status and uninstall commands though

Reply:
`m.initialize()` would fill the `DefaultOLMNamespace` value in manager's struct. So at least for `install` command, the `mgr.OLMNamespace` would always be `olm` when `InstallVersion` is called.